lib/xain.ex

```diff
@@ -200,4 +200,22 @@ defmodule Xain do
   defp set_defaults(attrs, name) do
     Keyword.merge(get_defaults(name), attrs)
   end
+
+  defp html_escape(html) do
```

Comment:
The escaped entities can be used for html injection, other entities such as an ampersand are less dangerous. An example of the danger of not escaping user input is embedding a script tag in the book description (when user supplied input can't be trusted): http://demo.exadmin.info/admin/products/8. The Phoenix.HTML library also contains functionality to properly escape html and generate html tags. Can also investigate if its possible to build upon this library if it is not a problem to add dependency on this library? (See the `to_iodata` for html escaping in https://github.com/phoenixframework/phoenix_html/blob/master/lib/phoenix_html/safe.ex).